doc/source/multivisit.rst

```diff
@@ -3,4 +3,535 @@
+set of parameters that can be used to control the sky cell definitions and IDs can be found in the
+:ref:`MVM Processing Code API page<multivisit_api`.
+
+Exposures in an input list are assumed to be in the current working directory when running the code,
+unless **input_path** has been provided which points to the location of the exposures to be processed.
+The return value **sky_cells_dict** is a dictionary where the keys are the names (labels) of each overlapping SkyCell
+and the value is the actual SkyCell object which contains the footprint and WCS (among other details) of the SkyCell.
+
+For example, observations from HST proposal 14175 were taken to study NGC 4594 (Sombrero Galaxy) using both the ACS and
+WFC3 cameras.  The footprints of all the HST/ACS and HST/WFC3 observations taken in this part of the sky as shown by MAST
+can be seen here:
+
+.. figure:: images/pg1202_acs_wfc3_mast_footprint.png
+  :width: 90%
+  :alt: Footprints of HST/ACS and HST/WFC3 observations of PG1202+281 and surrounding area.
+
+  Footprints of HST/ACS and HST/WFC3 observations of the the quasar PG1202+281 and surrounding area.
+
+There were 25 FLT and FLC exposures taken as part of this proposal making up these footprints.  The list of these
+files defined what was provided as **visit_input** to the `cell_utils.get_sky_cells()` function to get these SkyCell
+definitions:
+
+.. code:: python
+
+    sky_cells_dict = cell_utils.get_sky_cells(visit_input, input_path=None)
+    sky_cells_dict
+
+    {'skycell-p1889x07y19': SkyCell object: skycell-p1889x07y19,
+     'skycell-p1889x07y20': SkyCell object: skycell-p1889x07y20,
+     'skycell-p1970x15y03': SkyCell object: skycell-p1970x15y03,
+     'skycell-p1970x15y02': SkyCell object: skycell-p1970x15y02,
+     'skycell-p1970x16y02': SkyCell object: skycell-p1970x16y02}
+
+This indicates that these exposures overlap 5 SkyCells in 2 ProjectionCells **p1889** and **p1970** with the WCS defined in the
+defined SkyCell object for each SkyCell entry in this dictionary.  Each SkyCell object includes a list of all the
+exposures that overlap just that SkyCell, which can be used to generate those mosaics.  We can see that one SkyCell
+includes all exposures using:
+
+.. code:: python
+
+    for scell in sky_cells_dict:
+        print(scell, len(sky_cells_dict[sc].members))
+
+    skycell-p1889x07y19 25
+    skycell-p1889x07y20 8
+    skycell-p1970x15y03 8
+    skycell-p1970x15y02 17
+    skycell-p1970x16y02 8
+
+All subsequent examples will use the exposures for SkyCell **skycell-p1889x07y19**.  The member exposures filenames
+can be written out to its own input file named **skycell-p1889x07y19_mvm_input.txt** for MVM processing using:
+
+.. code:: python
+
+    with open('skycell-p1889x07y19_input.txt', 'w') as fout:
+        _ = [fout.write(f'{file}\n') for file in p1889_scells['skycell-p1889x07y19'].members]
+
```

Comment:
Should this be sky_cells_dict['skycell-p1889x07x19].members?

Reply:
Fixed in 3cd7f7e.